state.go

```diff
@@ -44,35 +42,35 @@ func (s state) validate() (bool, string) {
 
 	// settings
 	if (s.Settings == (config{}) || s.Settings.KubeContext == "") && !getKubeContext() {
-		return false, "ERROR: settings validation failed -- you have not defined a " +
+		return false, "settings validation failed -- you have not defined a " +
```

Comment:
maybe we should just make this return an `error` instead of `(bool, string)`

Reply:
that is a valid point, I've push commit with changes :-)